src/main/java/com/minecolonies/coremod/items/ItemClipBoard.java

```diff
@@ -17,16 +17,13 @@
 import net.minecraft.world.World;
 import org.jetbrains.annotations.NotNull;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.jetbrains.annotations.NotNull'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)